tests/cmake/find_package/bar.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+#include <foo.h>
+
+int main() {
```

Comment:
Perhaps comment here that we are explicitly using C++ streams here to ensure that the system include path find C++ headers first (maybe even link to the bug?)

Reply:
Sure.